lib/syntax_tools/src/erl_syntax_lib.erl

```diff
@@ -23,9 +23,6 @@
 %%
 %% This module contains utility functions for working with the
 %% abstract data type defined in the module {@link erl_syntax}.
-%%
-%% @type syntaxTree() = erl_syntax:syntaxTree(). An abstract syntax
```

Comment:
I agree that it is not the best solution to add aliases just for readability, but I still think that it is best to do so, since edoc produces the <code>(see module...)</code) thing. I think that the real solution would be to alter edoc so it produces a hyperlink like <code>module:type()</code> instead, but this is nothing I can prioritize now. Therefore I think that it would be good to use the aliases for the time being.